governance/README.md

```diff
@@ -0,0 +1,22 @@
+# Aave Governance Events
+
+## Description
+
+This bot monitors transactions for Aave governance events.
+
+## Supported Chains
+
+- Ethereum Mainnet
+
+## Alerts
+
+<!-- -->
+- AE-AAVE-GOVERNANCE
+  - Fired when a transaction log contains an event that matches a list of Aave governance events
+  - Severity is always set to "low"
+  - Type is always set to "suspicious"
+  - Metadata field contains Aave contract name and address, event name, and transaction hash
```

Comment:
This section needs to be updated with the list of alertIds and the associated finding information.  For example:
```
<!-- -->
- AE-AAVE-PROPOSAL-CREATED
  - Fired when a new Aave governance proposal is created
  - Severity is set to 'info'
  - Type is set to 'Info'
  - Metadata contains the Aave contract address and the proposal
```